jpf-symbc/src/main/gov/nasa/jpf/symbc/witness/Node.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Object that represents a single node of the violation witness
+ * It has two methods, serializeNode() and serializeAllNodes()
+ * Variable numberOfNode denotes the total number of node of violation witness
+ * Variable indexOfNode denotes the index of the node
+ * Variable noCounterExample is a flag that shows whether the counterexample is exists or not
+ */
+
+package gov.nasa.jpf.symbc.witness;
+
+import java.util.List;
+
+
+public class Node{
+    public int numberOfNode;
+
+    public int indexOfNode;
+    public boolean noCounterExample;
```

Comment:
Can you please write for each variable a comment on what it is about?

Reply:
@sohah , I wrote a comments for these variables. I'm removing this comment. Thank you!